app/assets/javascripts/admin/addon/adapters/api-key.js

```diff
@@ -1,13 +1,13 @@
 import RESTAdapter from "discourse/adapters/rest";
```

Comment:
I noticed that other import `RestAdapter` instead of `RESTAdapter`. Do you know if there is a difference between the two?

I believe this is just a naming thing because the default export is imported everytime and aliases, but it is just an inconsistency I noticed.